careers/students-and-graduates/index.html

```diff
@@ -0,0 +1,221 @@
+            </div>
+            <div class="media_body">
+                <h2>CFPB Summer Internship</h2>
+                <h3 class="h5">Look for applications here starting Fall 2015</h3>
+                <p class="short-desc">
+                    A twelve-week, paid summer internship program
+                    for high-potential students to assist
+                    in a variety of operational and mission-oriented projects.
+                    Students receive on-the-job training in the technical aspects
+                    of the organization's work.
+                    The students' interests, knowledge, and academic background
+                    play a large role in determining assignments.
+                </p>
+            </div>
+        </section>
+
+        <section class="media
+                        block
+                        block__padded-top
+                        block__border-top
+                        u-mb0">
+            <div class="media_image-container media_image-container__wide-margin">
+                <img class="u-hide-on-mobile"
+                     src="../../static/img/charts-round.png">
+            </div>
+            <div class="media_body">
+                <h2>Director's Financial Analyst</h2>
+                <h3 class="h5">Look for applications here starting Fall 2015</h3>
+                <p class="short-desc">
+                    Our highly selective, two-year rotational program.
+                    We seek out a small group of talented college graduates
+                    and challenge them to use their analytical skills
+                    to make financial markets work better for all.
+                </p>
+            </div>
+        </section>
+
+        <section class="media
+                        block
+                        block__padded-top
+                        block__border-top
+                        u-mb0">
+            <div class="media_image-container media_image-container__wide-margin">
+                <img class="u-hide-on-mobile"
+                     src="../../static/img/white-house-round.png">
+            </div>
+            <div class="media_body">
+                <h2>Presidential Management Fellow</h2>
+                <h3 class="h5">Applications due by XX/XX/XXXX</h3>
+                <p class="short-desc">
+                    The Federal government's flagship leadership development program
+                    at the entry level for advanced degree candidates.
+                    The two-year appointment provides a fast-paced opportunity
+                    to gain experience and develop talents.
+                    Fellows will be challenged with opportunities
+                    to flourish into problem solvers, strategic thinkers, and future leaders.
+                </p>
+                <p class="short-desc">
+                    <a href="http://www.pmf.gov/" class="jump-link jump-link__right">
```

Comment:
That's what I saw in the copy doc. @sarahsimpson08?
On Jun 26, 2015 7:22 AM, "Ans" <notifications@github.com> wrote:

> In careers/students-and-graduates/index.html
> <https://github.com/cfpb/cfgov-refresh/pull/680#discussion_r33350670>:
>
> > +                <img class="u-hide-on-mobile"
> > +                     src="../../static/img/white-house-round.png">
> > +            </div>
> > +            <div class="media_body">
> > +                <h2>Presidential Management Fellow</h2>
> > +                <h3 class="h5">Applications due by XX/XX/XXXX</h3>
> > +                <p class="short-desc">
> > +                    The Federal government's flagship leadership development program
> > +                    at the entry level for advanced degree candidates.
> > +                    The two-year appointment provides a fast-paced opportunity
> > +                    to gain experience and develop talents.
> > +                    Fellows will be challenged with opportunities
> > +                    to flourish into problem solvers, strategic thinkers, and future leaders.
> > +                </p>
> > +                <p class="short-desc">
> > +                    <a href="http://www.pmf.gov/" class="jump-link jump-link__right">
>
> Should this point to a PMF application page URL?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cfpb/cfgov-refresh/pull/680/files#r33350670>.
>